raster/r.out.vrml/r.out.vrml.html

```diff
@@ -28,7 +28,7 @@ <h2>NOTES</h2>
 This is a preliminary release of "<em>r.out.vrml</em>".
 
 For further information about VRML and available viewers for various
-platforms, see:<p><a href="http://www.w3.org/MarkUp/VRML/">VRML Virtual Reality Modeling Language</a>
+platforms, see:<p><a href="https://www.w3.org/MarkUp/VRML/">VRML Virtual Reality Modeling Language</a>
```

Comment:
But VRML was used here...

Reply:
I didn't expect a change, I just noted that it is not the current format used for this now that they changed to X3D. Maybe some work on it might be needed one day if someone really needs it (5+ years frame)